src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/render/MixinLivingEntityRenderer.java

```diff
@@ -65,22 +69,22 @@ private void injectRender(T livingEntity, float f, float g, MatrixStack matrixSt
     /**
      * Head rotation pitch injection hook
      */
-    @Redirect(method = "render(Lnet/minecraft/entity/LivingEntity;FFLnet/minecraft/client/util/math/MatrixStack;Lnet/minecraft/client/render/VertexConsumerProvider;I)V", at = @At(value = "INVOKE", target = "Lnet/minecraft/util/math/MathHelper;lerp(FFF)F", ordinal = 0))
-    private float injectRotationPitch(float g, float f, float s) {
+    @Redirect(method = "updateRenderState(Lnet/minecraft/entity/LivingEntity;Lnet/minecraft/client/render/entity/state/LivingEntityRenderState;F)V", at = @At(value = "INVOKE", target = "Lnet/minecraft/entity/LivingEntity;getLerpedPitch(F)F"))
+    private float injectRotationPitch(LivingEntity instance, float v) {
         final var rot = this.rotationPitch.get();
         if (rot != null) {
-            return MathHelper.lerp(g, rot.keyFloat(), rot.valueFloat());
+            return MathHelper.lerp(v, rot.keyFloat(), rot.valueFloat());
         } else {
-            return MathHelper.lerp(g, f, s);
+            return instance.getLerpedPitch(v);
         }
     }
 
 
-    @ModifyExpressionValue(method = "render(Lnet/minecraft/entity/LivingEntity;FFLnet/minecraft/client/util/math/MatrixStack;Lnet/minecraft/client/render/VertexConsumerProvider;I)V", at = @At(value = "INVOKE", target = "Lnet/minecraft/entity/LivingEntity;isInvisibleTo(Lnet/minecraft/entity/player/PlayerEntity;)Z"))
-    private boolean injectTrueSight(boolean original, T livingEntity) {
+    @ModifyExpressionValue(method = "render(Lnet/minecraft/client/render/entity/state/LivingEntityRenderState;Lnet/minecraft/client/util/math/MatrixStack;Lnet/minecraft/client/render/VertexConsumerProvider;I)V", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/render/entity/LivingEntityRenderer;isVisible(Lnet/minecraft/client/render/entity/state/LivingEntityRenderState;)Z"))
+    private boolean injectTrueSight(boolean original, @Local(argsOnly = true) S livingEntityRenderState) {
         // Check if TrueSight is enabled and entities are enabled or ESP is enabled and in glow mode
         if (ModuleTrueSight.INSTANCE.getRunning() && ModuleTrueSight.INSTANCE.getEntities() ||
-                ModuleESP.INSTANCE.getRunning() && ModuleESP.INSTANCE.requiresTrueSight(livingEntity)) {
+                ModuleESP.INSTANCE.getRunning() && ModuleESP.INSTANCE.requiresTrueSight(((LivingEntity) ((EntityRenderStateAddition) livingEntityRenderState).liquid_bounce$getEntity()))) {
             return false;
```

Comment:
```suggestion
            return true;
```
from 91d2b5831743e21de02b715e7996f3a33f86eab6

Reply:
Thank you.

Reply:
This is not fixed with that.